src/Workers/Worker.ts

```diff
@@ -43,7 +43,12 @@ const loadTrackables = async (msg: any) => {
     let src = msg.data;
     let refRows = msg.trackableHeight;
     let refCols = msg.trackableWidth;
-    let mat = new cv.matFromArray(refRows, refCols, cv.CV_8UC4, src);
```

Comment:
This wasn't correct, infact logging the `mat` it gives me an Unbinding error, though i have no errors in the console.